v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,683 @@
+// 2. 100 requests per 60 minutes - slow brute force - some attackers might try sidestepping the regular brute force detection by using a slower rate of requests
+const checkUserInSameNetwork = [{
+    keyStr: `${userIp}-${userEmail}`,
+    maxRequests: [
+        {
+            limit: 1,
+            perTimeIntervalMS: 1000,
+        },
+        {
+            limit: 100,
+            perTimeIntervalMS: 60 * 1000 * 60,
+        }
+    ]
+}]
+```
+
+
+```tsx
+const userIp = "127.0.0.1"; // this should be the user's IP address
+
+// Useful for limiting requests from the same network
+// This should usually have a higher number of requests/time interval allowed
+const checkNetwork = [{
+    keyStr: `${userIp}`,
+    maxRequests: [
+        {
+            limit: 100,
+            perTimeIntervalMS: 1000,
+        },
+    ]
+}]
+```
+
+```tsx
+const userEmail = "user@email.com"; // this should be the user's email
+
+// Useful for limiting requests for the user only
+const checkUserOnly = [{
+    keyStr: `${userEmail}`,
+    maxRequests: [
+        {
+            limit: 1,
+            perTimeIntervalMS: 1000,
+        },
+    ]
+}]
+```
+
+```tsx
+const userIp = "127.0.0.1"; // this should be the user's IP address
+const userEmail = "user@email.com"; // this should be the user's email
+
+// Checking by multiple keys at once
+const checkUserOnly = [
+    {
+        keyStr: `${userIp}-${userEmail}`,
+        maxRequests: [
+            {
+                limit: 1,
+                perTimeIntervalMS: 1000,
+            },
+            {
+                limit: 100,
+                perTimeIntervalMS: 60 * 1000 * 60,
+            }
+        ]
+    },
+    {
+        keyStr: `${userIp}`,
+        maxRequests: [
+            {
+                limit: 100,
+                perTimeIntervalMS: 1000,
+            },
+        ]
+    }
+]
+```
+
+- `passwordHash`: The SHA-1 hash of the password that needs to be checked against the breach database. If this is not provided, the password breach check will be skipped.
+- `email`: The email address that is being used for the authentication event. If this is not provided (or `phoneNumber`), the impossible travel detection, new device detection and device count detection will be skipped.
+- `phoneNumber`: The phone number that is being used for the authentication event. If this is not provided (or `email`), the impossible travel detection, new device detection and device count detection will be skipped.
+- `actionType`: The type of action that is being performed. The possible values are:
+      - "emailpassword-sign-in"
+      - "emailpassword-sign-up"
+      - "send-password-reset-email"
+      - "passwordless-send-email"
+      - "passwordless-send-sms"
+      - "totp-verify-device"
+      - "totp-verify-totp"
+      - "thirdparty-login"
+      - "emailverification-send-email"
+
+
+Below is an example of how to make the request to the anomaly detection endpoint:
+
+```tsx
```

Comment:
i dont think this section is needed cause the seciton below will have this anyway